prow/plugins.yaml

```diff
@@ -206,7 +202,10 @@ slack:
         release-1.14:
             - aleksandra-malinowska # 1.14 patch release team
             - tpepper # 1.14 patch release team
-            - hoegaarden # 1.14 branch manager
+            - hoegaarden # 1.14 patch release team
+        release-1.15:
```

Comment:
https://github.com/kubernetes/sig-release/blob/master/releases/release-1.15/release_team.md

Reply:
thanks! we are not sure about other shadows yet, I will follow up with this :)